lib/model/model/mock/MockArticle.dart

```diff
@@ -52,9 +52,16 @@ class MockArticle extends MockEntity<ArticleEntity> {
     return entity;
   }
 
+  ArticleEntity buildStageFromArticle(ArticleEntity article) {
+    final entity = article;
+    entity.images = null;
+    return entity;
+  }
```

Comment:
Created method to remove images given an Article entity

Reply:
this has made me think, the entity fields should all be final really, and we should do this with a special transformer.

Reply:
But for now this is fine